pkg/controller/import-controller.go

```diff
@@ -410,12 +410,15 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 		delete(anno, cc.AnnBoundConditionReason)
 	}
 
+	if pvc.GetLabels() == nil {
+		pvc.SetLabels(make(map[string]string, 0))
+	}
 	if !checkIfLabelExists(pvc, common.CDILabelKey, common.CDILabelValue) {
-		if pvc.GetLabels() == nil {
-			pvc.SetLabels(make(map[string]string, 0))
-		}
 		pvc.GetLabels()[common.CDILabelKey] = common.CDILabelValue
 	}
+	if cc.IsPVCComplete(pvc) {
```

Comment:
hopefully, there's not a similar problem in the import controller as with the import populator where this will not be executed on an update conflict requeue

Reply:
Let's see what CI says on the latest rev

Reply:
Reading through the code I am expecting the same issue